client/src/plugins/zeebe-plugin/connection-manager-plugin/__tests__/ConnectionManagerPluginSpec.js

```diff
@@ -43,6 +43,7 @@ describe('ConnectionManagerPlugin', function() {
           activeTab: DEFAULT_ACTIVE_TAB
         });
       }
+      return { cancel: () => {} };
```

Comment:
Perhaps we could extract this into a `createSubscribe` helper or something, but it's not mandatory.